tests/foreman/ui/test_discoveryrule.py

```diff
@@ -208,9 +217,17 @@ def test_positive_list_host_based_on_rule_search_query(
     discovered_host = target_sat.api_factory.create_discovered_host(
         ip_address=ip_address, options={'physicalprocessorcount': cpu_count}
     )
-    # create an other discovered host with an other cpu count
```

Comment:
Why did we create second host earlier? This was not part of the test scenario.

Reply:
Upon checking, I discovered the second host is present for more than 5 years and it was added at that time. No one cared why we added a second host, so we're removing it now